letsencrypt/config.yaml

```diff
@@ -120,6 +120,7 @@ schema:
     sakuracloud_api_token: str?
     transip_api_key: str?
     transip_username: str?
+    transip_global_key: str?
```

Comment:
Probably better to define as `list(yes|no)?`

Reply:
Good point, fixed